EllisWeb.Gematria/GematriaOptions.cs

```diff
@@ -0,0 +1,67 @@
+﻿namespace EllisWeb.Gematria
+{
+    /// <summary>
+    /// Options when converting a number into its Gematria representation
+    /// </summary>
+    public class GematriaOptions
+    {
+        /// <summary>
+        /// default value for <see cref="IncludeSeparators"/>
+        /// </summary>
+        public const bool DEFAULT_SHOULD_INCLUDE_SEPARATORS = true;
+
+        /// <summary>
+        /// default value for <see cref="ThousandsSeparator"/>
+        /// </summary>
+        public const char DEFAULT_THOUSANDS_SEPARATOR = '\'';
+
+        /// <summary>
+        /// default value for <see cref="TensSeparator"/>
+        /// </summary>
+        public const char DEFAULT_TENS_SEPARATOR = '"';
+
+        /// <summary>
+        /// default value for <see cref="AddQuoteAfterSingleChar"/>
+        /// </summary>
+        public const bool DEFAULT_ADD_QUOTE_AFTER_SINGLE_CHAR = true;
```

Comment:
default should be set on relevant line

Reply:
and remove the const?
in that case, we are forcing the user to always send all parameters to `GematriaOptions` ctor.